File: src/main/java/UserDTO.java
import org.json.simple.JSONObject;

/**
 * Created by Marian on 10/24/2015.
 */
public class UserDTO {
    private String name;
    private Long score;

    public Long getScore() {
        return score;
    }

    public void setScore(Long score) {
        this.score = score;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return this.getName() + " " + this.getScore();
    }
}

File: MPS Project/Assets/3DDiceRoller/Utils/UserDTO.cs
﻿using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

[JsonObject(MemberSerialization.OptIn)]
public class UserDTO
{
    [JsonProperty]
    private String name;

    [JsonProperty]
    private Int32 score;

    public UserDTO() { }

    [JsonConstructor]
    public UserDTO(String name, Int32 score)
    {
        this.score = score;
        this.name = name;
    }

    public void SetScore(Int32 score)
    {
        this.score = score;
    }

    public Int32 GetScore()
    {
        return this.score;
    }

    public void SetName(string name)
    {
        this.name = name;
    }

    public string GetName()
    {
        return this.name;
    }

}
